packages/field-plugin/src/createFieldPlugin/createFieldPlugin.ts

```diff
@@ -47,12 +47,11 @@ export const createFieldPlugin: CreateFieldPlugin = ({
     return () => undefined
   }
 
-  const { uid } = params
+  const { uid, secure, host } = params
+  const origin = `${secure ? 'https://' : 'http://'}${host}`
```

Comment:
Hey @eunjae-lee, I was thinking here and using the `${host}` variable directly maybe not strict/secure enough to avoid intercepting data between the field-plugin and the Storyfront because the `host` can be anything, not only `https://app.storyblok.com/` or `https://plugin-sandbox.storyblok.com`.

An example would be using our `hosted-plugin` (or any other plugin that intercepts the calls between the field-plugin and the storyfront) since it can pass any `host` as a query-parameter when adding the iframe `src` attribute.

So, I may be wrong since I don't fully understand all this behavior but something like it may be used to intercept values between the `real field plugin` and the `storyfront`.

![Screenshot 2024-01-11 at 14 43 55](https://github.com/storyblok/field-plugin/assets/1240591/a08d7349-3b70-4a26-8f5d-eee11f17eb56)

May the origin should be restricted to only `https://app.storyblok.com/` or `https://plugin-sandbox.storyblok.com` but if we do that our `hosted-plugin` would stop working, I suppose.

Example:
![image](https://github.com/storyblok/field-plugin/assets/1240591/383819b8-1f05-4148-95fd-ee2409422eaf)

This way, I think no one would be able to intercept the message exchanges between Storyfront and the real field plugin.

Do you think my concern makes sense, @eunjae-lee? 🤔 
I may took the wrong way of seeing this 🤣

Reply:
Yeah that's a good point 👍🏼 I cannot imagine how a malicious actor manipulate this origin and do malicious acts, but having this allow-list should be much better than not.